pycontrails/core/flight.py

```diff
@@ -1386,11 +1426,57 @@ def _verify_altitude(altitude: np.ndarray, nominal_rocd: float, freq: np.timedel
         )
 
 
-def _dt_waypoints(
+def filter_altitude(
```

Comment:
Long term this should be replaced by scipy smoothing filter (See traffic `core.flight.filter`)